app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,115 @@
+<h1 class="text-center mb-2 text-xl font-extrabold text-gray-900  md:text-2xl lg:text-3xl">
+  <%= @recipe.name %>
+</h1>
+<div class="flex justify-center">
+<%= image_tag ("recipes.jpg"), :class => "w-6/12 border rounded-3xl"%>
+</div>
+<div class="flex flex-col gap-y-1">
+  <div class="flex justify-between">
+    <div class="text-xl font-semibold text-gray-900">
+      <span>Preparation Time: </span> <span class="italic"><%= @recipe.preparation_time %></span>
+    </div>
+    <%= form_for @recipe, method: :PATCH do |f| %>
+      <%= f.submit "Public", class: 'text-white bg-blue-700 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm w-36 sm:w-auto px-3 py-2 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800' if @recipe.public %>
+      <%= f.submit "Private" , class: 'text-white bg-blue-700 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm w-36 sm:w-auto px-3 py-2 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800' unless @recipe.public %>
+    <% end %>
+  </div>
+  <div class="text-lg font-semibold text-gray-900">
+    <span>Cooking Time: </span> <span class="italic"><%= @recipe.cooking_time %></span>
+  </div>
+  <div>
+    <span class="text-lg font-semibold text-gray-900">Steps :</span>
+    <span><%= @recipe.description %></span>
+  </div>
+  <%# modal and its button %>
+  <% if can? :destroy, @recipe %>
+    <div class="flex justify-between my-2">
+      <!-- Modal toggle -->
+      <button data-modal-target="defaultModal" data-modal-toggle="defaultModal" class="block text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800" type="button">
```

Comment:
- [ ] I feel why the modal does not pop up on initial render should be a bootstrap issues, kindly check your bootstrap version and make sure you used the correct modal component :+1: